cpp/CMakeLists.txt

```diff
@@ -82,8 +85,34 @@ set(GPU_ARCHS "" CACHE STRING
     Pass 'ALL' if you want to compile for all supported GPU architectures.
     Empty string means to auto-detect the GPUs on the current system")
 
-set(CMAKE_IGNORE_PATH "${CMAKE_INSTALL_DIR}/lib" CACHE STRING
-    "Ignore any libs added implicitly from the CMAKE_INSTALL_DIR")
+
+##############################################################################
+# - Conda environment detection ----------------------------------------------
+
+if(CONDA_BUILD)
+  if(DEFINED ENV{CONDA_PREFIX})
+    set(CMAKE_PREFIX_PATH "$ENV{CONDA_PREFIX};${CMAKE_PREFIX_PATH}")
+    set(CONDA_INCLUDE_DIRS "$ENV{CONDA_PREFIX}/include")
+    set(CONDA_LINK_DIRS "$ENV{CONDA_PREFIX}/lib")
```

Comment:
Are we supposed to be using these variables inside cmake flow?

Reply:
Removed them for now since we don't need them unless the CMAKE_PREFIX_PATH doesn't fix it